apps/openmw/mwmechanics/aicombataction.cpp

```diff
@@ -230,18 +233,48 @@ namespace MWMechanics
             }
         }
 
+
+        // New Spell Random Priority Generator.
```

Comment:
This line only makes sense in the context of this pull request. I suggest to remove it and the line 277 as well..